Jenkinsfile

```diff
@@ -3,7 +3,7 @@ library "jenkinsLibrary@develop"
 buildPublishPipeline(
     slackChannel: "sandbox-github", 
     asProjectRepo: "OMJSON",
-    asProjectVersion: "main", 
+    asProjectVersion: "feature/lpm-dependency-update", 

```

Comment:
Just making a note here that this needs to get changed back before merging. Maybe there is a better way?

Reply:
I don't know a better way, but yes, this will get reverted before the merge.